pages/docs/security.mdx

```diff
@@ -0,0 +1,114 @@
+from langfuse.decorators import observe, langfuse_context
+from langfuse.openai import openai # OpenAI integration
+from llm_guard.output_scanners import Deanonymize
+
+vault = Vault()
+
+prompt = """
+Plaintiff, Jane Doe, by and through her attorneys, files this complaint against Defendant, Big Corporation, and alleges upon information and belief, except for those allegations pertaining to personal knowledge, that on or about July 15, 2023, at the Defendant's manufacturing facility located at 123 Industrial Way, Springfield, Illinois, Defendant negligently failed to maintain safe working conditions, leading to Plaintiff suffering severe and permanent injuries. As a direct and proximate result of Defendant's negligence, Plaintiff has endured significant physical pain, emotional distress, and financial hardship due to medical expenses and loss of income. Plaintiff seeks compensatory damages, punitive damages, and any other relief the Court deems just and proper."""
+
+@observe()
+def anonymize(input: str):
+  scanner = Anonymize(vault, preamble="Insert before prompt", allowed_names=[], hidden_names=["Test LLC"],
+                    recognizer_conf=BERT_LARGE_NER_CONF, language="en")
+  sanitized_prompt, is_valid, risk_score = scanner.scan(prompt)
+  return sanitized_prompt
+
+@observe()
+def deanonymize(sanitized_prompt: str, answer: str):
+  scanner = Deanonymize(vault)
+  sanitized_model_output, is_valid, risk_score = scanner.scan(sanitized_prompt, answer)
+
+  return sanitized_model_output
+
+@observe()
+def summarize_transcript(prompt: str):
+  sanitized_prompt = anonymize(prompt)
+
+  answer = openai.chat.completions.create(
+        model="gpt-3.5-turbo",
+        max_tokens=100,
+        messages=[
+          {"role": "system", "content": "Summarize the given court transcript."},
+          {"role": "user", "content": sanitized_prompt}
+        ],
+    ).choices[0].message.content
+
+  sanitized_model_output = deanonymize(sanitized_prompt, answer)
+
+  return sanitized_model_output
+
+@observe()
+def main():
+    return summarize_transcript(prompt)
+
+main()
+```
+![Trace](../../public/images/docs/security-docs-trace.png)
+
+<Callout type="info">
+  See the trace generated from this example. You can track each step--Anonymize, Summarize, and Deanonymize-- separately: https://cloud.langfuse.com/project/clvh71wlt0000wpuqhferayy5/traces/d31a6a8a-03b4-4487-8d87-40c3a68464a5?observation=aa28105d-c33e-4333-828a-d916de48e387
+</Callout>
+
+
+
+## Advanced Usage
+
+### Langfuse Model-Based Evaluations
+
+You can use a security tool like LLM Guard to scan model prompts and completions, and then use Langfuse to track and evaluate this data after the call is executed.
+
+One way is to understand which flags (toxicity, bias, relevance) are set off the most, which can help teams identify which security risks are most pressing and build more robust tools against those specific issues.
```

Comment:
Basically the model based eval scores give you a pointer where to look more closely and where you may need to fine tune your security set-up. if you have a lot of traffic it's impractical to look at everything yourself so you need to narrow the sample